.eslintrc

```diff
@@ -7,6 +7,10 @@
     "mocha"
   ],
   "rules": {
-    "mocha/no-exclusive-tests": "error"
+    "mocha/no-exclusive-tests": "error",
+    "indent": ["error", 2, {
+      "ArrayExpression": "first",
+      "ObjectExpression": "first"
+    }]
```

Comment:
It will probably make sense to move this to the shared "hypothesis" config in future, but for the time being I've left it here.